templates/internet-of-things/index.html

```diff
@@ -324,216 +261,231 @@ <h2>Ready&ndash;to&ndash;go gateway solutions</h2>
-        {{
-          image(
-            url="https://assets.ubuntu.com/v1/dca2e4c4-raspberry-logo.png",
-            alt="",
-            width="195",
-            height="65",
-            hi_def=True,
-          ) | safe
-        }}
-      </header>
-      <hr class="u-sv1 u-hide--small" />
-      <h3 class="p-card__title">Raspberry Pi 2, 3 and 4</h3>
-      <p class="p-card__content">For fun, for education and for profit, the RPi makes device development personal and entertaining. With support for both the Pi2, Pi3 and the new Pi4, Ubuntu Core supports the world's most beloved board.</p>
-      <p class="p-card__content"><a href="https://www.raspberrypi.org/">Learn more on rasberrypi.org&nbsp;&rsaquo;</a></p>
-    </div>
-  </div>
-  <div class="row u-equal-height">
-    <div class="col-6 p-card">
-      <header class="no-border u-align--center u-vertically-center u-hide--medium u-hide--small" style="min-height: 7rem;">
-        {{
-          image(
-            url="https://assets.ubuntu.com/v1/a8d80d5a-Qualcomm-Logo.svg",
-            alt="",
-            width="200",
-            height="65",
-            hi_def=True,
-          ) | safe
-        }}
-      </header>
-      <hr class="u-sv1 u-hide--small" />
-      <h3 class="p-card__title">Qualcomm Dragonboard</h3>
-      <p class="p-card__content">64-bit ARM boards bring high-end performance to the low-power segment, enabling a new class of drones and mobile intelligence.</p>
-      <p class="p-card__content"><a href="https://developer.qualcomm.com/hardware/dragonboard-410c">Learn more on qualcomm.com&nbsp;&rsaquo;</a></p>
-    </div>
-    <div class="col-6 p-card">
-      <header class="no-border u-align--center u-vertically-center u-hide--medium u-hide--small" style="min-height: 7rem;">
-        {{
-          image (
-          url="https://assets.ubuntu.com/v1/327b8104-AMD_E_Blk_RGB.png",
-          alt="",
-          width="271",
-          height="65",
-          hi_def=True,
-          loading="auto|lazy"
-          ) | safe
-        }}
-      </header>
-      <hr class="u-sv1 u-hide--small" />
-      <h3 class="p-card__title">Xilinx Evaluation Kits and SOMs</h3>
-      <p class="p-card__content">Zynq&reg; UltraScale+&trade; MPSoCs integrate a quad-core 64-bit Arm application processor with real-time processors and FPGA logic to offer an embedded, low-latency acceleration platform.</p>
-      <p class="p-card__content"><a href="https://www.xilinx.com/applications/industrial/robotics.html#solutionStack?utm_source=canonical&utm_medium=referral&utm_campaign=ubuntu&utm_content=iot&utm_term=eval_kit">Learn more on xilinx.com&nbsp;&rsaquo;</a></p>
+    <div class="col-6">
+      <div class="p-logo-section has-misaligned-images">
+        <div class="p-logo-section__items u-align--center">
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/6079d02b-amd-logo.png",
+              alt="",
```

Comment:
Same here

Reply:
I think due to the complexity of the image and lack of imformation it adds, we can omit the alt text on this one

Reply:
This is a logo though, we always add alt text for the logos https://accessibleweb.com/question-answer/logos-need-follow-accessibility-guidelines/

Reply:
Sorry, I thought you were referring to a different image